filebeat/harvester/reader/reader.go

```diff
@@ -1,42 +1,42 @@
-package harvester
+package reader
 
 import (
 	"errors"
 	"io"
 	"os"
 	"time"
 
-	"github.com/elastic/beats/filebeat/input"
+	"github.com/elastic/beats/filebeat/harvester/source"
+	"github.com/elastic/beats/filebeat/input/file"
 	"github.com/elastic/beats/libbeat/logp"
 )
 
 type logFileReader struct {
```

Comment:
My current intention is to group all readers together so each harvester could make use of the readers.